src/asciidoc/docs/develop/testingTemplates.adoc

```diff
@@ -185,3 +186,232 @@ If you publish your template as *Unlisted*, you can test it under exactly the sa
+In most cases, selecting one of the contexts in the debugger displays an events pane which shows the events associated with that context.
+An events pane allows you to see in real time the events that are generated as you interact with the template in the selected context.
+Events generated by the template are listed on the right side of the events pane.
+Events generated by the Koji platform are listed on the left side.
+
+The info for each event includes:
+
+* The type of event – PLATFORM or TEMPLATE
+* The name of the event – For example: *Checked*
+* Properties set by the event (optional) – This may include descriptions of the properties.
+* New values for the properties (optional) – The new values for the properties set by the event.
+
+If you open a template and select the *Remix* context, here's an example of some events that you might see in the events pane:
+
+* TEMPLATE +
+Ready
+
+* PLATFORM +
+Changing remix state +
+VALUE +
+Remixing
+
+* TEMPLATE +
+Set value +
+PATH +
++++[general, title]+++ +
+NEW VALUE +
++++""+++
+
+If you click in an editable field on the remixing page, a *Clicked* event will appear in the event pane.
+For example:
+
+* TEMPLATE +
+Clicked +
+POSITION +
+(131, 583)
+
+If you then change the value of the field you clicked, a *Set Value* event will appear.
+For example:
+
+* TEMPLATE +
+Set value +
+PATH +
++++[general, goal]+++ +
+NEW VALUE +
+"21.00" +
+TYPE +
+Skip update
+
+=== Info and Features
+
+You can view and modify environment info for your template, as well as settings and data for implemented features.
+If you click on a feature that is not implemented, a message will be displayed saying that no data was found for the feature.
+
+The *Info and Features pane* includes the following panels.
+
+==== Environment info
+
+The *Environment info* panel displays your templates's *App Id*, *Frame URL*, and *VCC editor source*.
+You can click *Modify* to edit the frame URL.
+
+The *Environment info* panel also allows you to edit the *VCC editor source*.
+When you publish a template, the VCC editor data defines the VCCs available in the template.
+So when the template calls something like `InstantRemixing.onPresentControl(['general', 'photo'])`, the editor looks in the data for the `general.photo` path and knows to show the image VCC.
+If you're working on a new version of a template that uses different VCCs from the currently published template, and you're using the debugger to override the URL and debug something running locally or running in the web IDE, you can edit the *VCC editor source* to override the live data.
+
+You edit the *VCC editor source* by clicking *Modify* and selecting *Override and use different VCC editor data*.
+This displays the data in JSON format.
+You can directly edit the JSON code to override the live values.
+
+For example, suppose the VCC editor data for a template reads as follows:
```

Comment:
The VCC editor data seems like too much detail for this page. Someone who’s is at this point should already know about VCC data. Your example use case – switching from an `image` type to a `video` type – is very useful and should be the highlighted part. Consider removing the rest of the JSON and jumping straight into the use case.